src/schema/v2/artwork/index.ts

```diff
@@ -1515,10 +1519,22 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
             _.sortBy(typedImages, "position")
           )
 
-          const typedVideos = VIDEOS[id]
-            ? [{ ...VIDEOS[id], type: "Video" }]
+          const typedVideos = external_video_id
```

Comment:
#minor since this intentionally just supports a single video, maybe this should be `const typedVideo` and be an object (vs. an array) - doesn't need to be spread below.

Reply:
Will fix 👍